compiler/src/dotty/tools/dotc/config/CliCommand.scala

```diff
@@ -141,26 +141,34 @@ trait CliCommand:
   /** Used for the formatted output of -Xshow-phases */
   protected def phasesMessage(using ctx: Context): String =
   
+    import scala.io.AnsiColor.*
```

Comment:
Should use the tools in `dotty.tools.dotc.printing.Highlighting`.

Reply:
I was going to ditch the format string anyway as harder to construct than manual padding, but now I will ditch it because formatter field widths don't account for ansi escapes. In this case, widths could be adjusted to accommodate the escapes, but in general it might trim them.